src/quacc/calculators/qchem.py

```diff
@@ -199,3 +214,27 @@ def read_results(self):
                 struct.unpack("d", binary[ii * 8 : (ii + 1) * 8])[0]
                 for ii in range(len(binary) // 8)
             )
+        if self.job_type == "freq":
+            tmp_hess_data = []
+            # Read Hessian scratch file in 8 byte chunks
+            with zopen("132.0", mode="rb") as file:
+                binary = file.read()
+                for ii in range(int(len(binary) / 8)):
```

Comment:
Are you sure you want to use `int` here and not `round`? Note that `int(4.7)` goes to 4, not 5.